pkg/network/status/status.go

```diff
@@ -280,9 +262,9 @@ func (m *StatusProber) Start(done <-chan struct{}) {
 	}()
 }
 
-// CancelVirtualServiceProbing cancels probing of the provided VirtualService.
-func (m *StatusProber) CancelVirtualServiceProbing(vs *v1alpha3.VirtualService) {
-	key := fmt.Sprintf("%s/%s", vs.Namespace, vs.Name)
+// CancelIngressProbing cancels probing of the provided Ingress
+func (m *StatusProber) CancelIngressProbing(ing *v1alpha1.Ingress) {
```

Comment:
Can we make the signature of this suitable for direct use with the informer callback (since that's it's intended usage)?

Reply:
Done